### DISCUSSION TITLE
Pod death failures

### DISCUSSION THREAD

**Discussion by saaramitay** [dagster-io/dagster#18621] [Q&A]
Hi,

Sometimes the pod running a Dagster pipeline might fail in the middle of the run because of some issues related to the infrastructure.

Is it possible to configure a way to detect such failures and spin up a new pod to continue the same run? (the run should keep the succeeded ops results and data)

**Comment by jamiedemaria** [comment]
You could look into enabling [run retries](https://docs.dagster.io/deployment/run-retries#run-retries). If you are running jobs that materialize assets, you could also use a sensor to detect these job failures, then return a `RunRequest` for that same job with the `stale_assets_only` parameter set to `True`. This will only materialize the assets in that job that are considered "stale"

**Reply by saaramitay** [reply]
@jamiedemaria Thanks for replying.

We enable configuration from the helm chart of dagster.
Will run retries always rerun from the begging of the run for every failure or it will only rerun from the begging of the run for failures related to the pod?
